src/Sylius/Bundle/CoreBundle/Fixture/Factory/PaymentMethodExampleFactory.php

```diff
@@ -87,7 +90,8 @@ public function create(array $options = [])
             $paymentMethod->setCurrentLocale($localeCode);
             $paymentMethod->setFallbackLocale($localeCode);
 
-            $paymentMethod->setName(sprintf('[%s] %s', $localeCode, $options['name']));
+            $paymentMethod->setName($options['name']);
```

Comment:
This may create a lot of confusion if someone will change his locale and translatables will be the same. We can just omit printing locale code for default locale, wdyt?